### ISSUE TITLE
Palm Rejection - Surface Laptop 2

### ISSUE THREAD

**Issue by dbranham20** [jakeday/linux-surface#423]
Hi,

Sorry if this does not relate to the kernel patches, but I posted anyways because I wasn't sure. 

Is there something that can be done about the lack of palm rejection on the Surface Laptop 2's touchpad? For example, typing with the keyboard can sometimes be very annoying because the touchpad picks up and registers every bit of touch to it, even when not intended. 

If this is not the right place for this discussion, please lead me in the right direction!

Thanks,

**Comment by TheTetrabyte** [comment]
Curious about this as well, really the only thing I've noticed that does not work well.

**Comment by nkkollaw** [comment]
To me, it's a Linux problem.

I go into my DE's settings and pick "disable touchpad while typing", then set a few milliseconds as a delay to re-enabled it. You adjust the settings according to your typing speed.

**Comment by dbranham20** [comment]
I turned that on and it seems to help a little, but Linux Mint (with Cinnamon) doesn't give me an option to set a delay

**Comment by nkkollaw** [comment]
Yes, they probably think their algorithm is good enough. I'm using Xfce and there is no feature to make touchpads even remotely usable without manually setting a delay, but they do let you set it.

We have to wait for this guy: https://bill.harding.blog/2019/03/25/linux-touchpad-like-a-macbook-progress-and-a-call-for-help/
